tests/ui-testing/playwright-tests/remotepipeline.spec.js

```
@@ -0,0 +1,514 @@
+    .locator(
+      `[data-test="pipeline-list-${randomPipelineName}-delete-pipeline"]`
+    )
+    .click();
+  await page.locator('[data-test="confirm-button"]').click();
+}
+test.describe("Pipeline testcases", () => {
+  // let logData;
+  function removeUTFCharacters(text) {
+    // console.log(text, "tex");
+    // Remove UTF characters using regular expression
+    return text.replace(/[^\x00-\x7F]/g, " ");
```

Biome (1.9.4)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)